task_2/sergii.garanenko/build.gradle

```diff
@@ -0,0 +1,14 @@
+apply plugin: 'java'
+apply plugin: 'war'
+apply from: 'https://raw.github.com/akhikhl/gretty/master/pluginScripts/gretty.plugin'
+
+repositories{
+    jcenter()
+}
+
+dependencies{
+    compile group: 'javax.servlet', name: 'javax.servlet-api', version: '3.1.0'
+    // https://mvnrepository.com/artifact/org.assertj/assertj-core
```

Comment:
It is not a good practice to leave dead code. If you need you can always look for it in the previous commit in the git.